worlds/sc2/MissionTables.py

```diff
@@ -18,6 +18,32 @@ class MissionPools(IntEnum):
     FINAL = 5
 
 
+class MissionFlags(IntFlag):
```

Comment:
Class should be named in singular (like we have `SC2Mission`)

Reply:
Huh, I've always thought of int flags like collections, but looks like the standard across Archipelago is to make them singular. Will fix in a bit.